objax/optimizer/__init__.py

```diff
@@ -16,3 +16,4 @@
 from .ema import *
 from .momentum import *
 from .sgd import *
+from .lars import *
```

Comment:
Alphabetical order please.

Reply:
done

Reply:
Hmm, maybe you didn't commit your changes because it doesn't appear to be done.

Reply:
should be in the latest push